scripts/cybershake/run_cybershake.py

```diff
@@ -243,6 +243,8 @@ def main():
     parser.add_argument(
         "config_file",
         help="The location of the config file containing everything to be run",
+        nargs="?",
+        default="task_config.yaml",
```

Comment:
This will look for task_config.yaml in the current working directory (Where the script was called from) rather than the directory the file is in.
Check management/install_mgmt_db for an example of how to reference a file in the same directory.